README.md

```diff
@@ -110,15 +110,49 @@ namespace FileUploader
 ```
 
 ## Running Minio Client Examples
+#### On Windows 
 * Clone this repository.
 
-* Build the project to produce the Minio.Examples console app.
+* Build the project in Visual Studio to produce the Minio.Examples console app.
 
 * Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
   Uncomment the example test cases such as below in Program.cs to run an example.
 ```cs
   //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
 ```
+* Run the Minio.Client.Examples.NET452 or Minio.Client.Examples.NetCore project from Visual Studio
+#### On Linux (Ubuntu 16.04 and above)
+* Clone this repository.
+
+* Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
+  Uncomment the example test cases such as below in Program.cs to run an example.
+```cs
+  //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
+```
+<blockquote> NOTE: minio-dotnet requires mono 5.0.1 stable release and .NET Core 1.0 SDK to build on Linux. </blockquote>
+```
+bash $ git clone https://github.com/minio/minio-dotnet && cd minio-dotnet 
+$ sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
```

Comment:
These are standard mono and .netcore installation steps and here to illustrate how to setup for current edition of Mono/Netcore, do we want to keep a script that will have to change in step with Mono/.Netcore version updates?